llvm/test/CodeGen/AIE/aie2/schedule/vlda_vldb.mir

```diff
@@ -51,3 +52,42 @@ body:             |
     $wl1 = VLDB_128_ag_idx $p0, $dj0
     $wl2 = VLDA_128 $p0
 ...
+
+---
+name:            VLDA_VLDB_with_MMO
+alignment:       16
+body:             |
+  bb.0.entry:
+    ; CHECK-LABEL: name: VLDA_VLDB_with_MMO
+    ; CHECK: $wl1 = VLDB_dmw_ldb_ag_idx_imm $p0, 0 :: (load (<8 x s32>))
+    ; CHECK-NEXT: $wl2 = VLDA_dmw_lda_w_ag_idx_imm killed $p0, 0 :: (load (<8 x s32>))
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    $wl1 = VLDB_dmw_ldb_ag_idx_imm $p0, 0 :: (load (<8 x s32>))
+    $wl2 = VLDA_dmw_lda_w_ag_idx_imm $p0, 0 :: (load (<8 x s32>))
+...
+
+# We expect all the load/store to be with MMOs.
+---
+name:            VLDA_VLDB_withOut_MMO
+alignment:       16
+body:             |
+  bb.0.entry:
+    ; CHECK-LABEL: name: VLDA_VLDB_withOut_MMO
+    ; CHECK: BUNDLE implicit-def $wl2, implicit-def $wl1, implicit killed $p0 {
+    ; CHECK-NEXT:   $wl2 = VLDA_dmw_lda_w_ag_idx_imm $p0, 0
+    ; CHECK-NEXT:   $wl1 = VLDB_dmw_ldb_ag_idx_imm killed $p0, 0
+    ; CHECK-NEXT: }
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    $wl1 = VLDB_dmw_ldb_ag_idx_imm $p0, 0
+    $wl2 = VLDA_dmw_lda_w_ag_idx_imm $p0, 0
```

Comment:
I think this is confusing, in absence of MMO, we should block all the banks. This goes in the direction of my comment here: https://github.com/Xilinx/llvm-aie/pull/95/commits/b2d99ac48855c1ebf1cb1574df7dbc86239be7af#r1711136944